docs/vcpkg-schema-definitions.schema.json

```diff
@@ -0,0 +1,715 @@
+    },
+    "override": {
+      "description": "A version override.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "$ref": "#/definitions/identifier"
+        },
+        "version": {
+          "$ref": "#/definitions/version-text"
+        },
+        "port-version": {
+          "$ref": "#/definitions/port-version"
+        }
+      },
+      "patternProperties": {
+        "^\\$": {}
+      },
+      "required": [
+        "name",
+        "version"
+      ]
+    },
+    "packages-list": {
+      "type": "object",
+      "description": "A list of port names which a port registry is used to fetch.",
+      "properties": {
+        "packages": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/identifier"
+          }
+        }
+      },
+      "required": [
+        "packages"
+      ]
+    },
+    "package-name": {
+      "type": "string",
+      "description": "Name of a package.",
+      "allOf": [
+        {
+          "description": "Package name must be a dot-separated list of valid identifiers",
+          "pattern": "^[a-z0-9]+(-[a-z0-9]+)*(\\.[a-z0-9]+(-[a-z0-9]+)*)*$"
+        },
+        {
+          "not": {
+            "description": "Identifiers must not be a Windows filesystem or vcpkg reserved name.",
+            "pattern": "(^|\\.)(prn|aux|nul|con|lpt[1-9]|com[1-9]|core|default)(\\.|$)"
+          }
+        }
+      ]
+    },
+    "platform-expression": {
+      "description": "A specification of a set of platforms. See https://github.com/microsoft/vcpkg/blob/master/docs/specifications/manifests.md#definitions.",
+      "type": "string"
+    },
+    "port-version": {
+      "description": "A non-negative integer indicating the port revision. If this field doesn't exist, it's assumed to be `0`.",
+      "type": "integer",
+      "minimum": 0,
+      "default": 0
+    },
+    "registry-array": {
+      "oneOf": [
+        {
+          "allOf": [
+            {
+              "oneOf": [
+                {
+                  "$ref": "#/definitions/builtin-registry"
+                },
+                {
+                  "$ref": "#/definitions/git-registry"
+                },
+                {
+                  "$ref": "#/definitions/filesystem-registry"
+                }
+              ]
+            },
+            {
+              "$ref": "#/definitions/packages-list"
+            }
+          ]
+        },
+        {
+          "$ref": "#/definitions/artifact-registry"
+        }
+      ]
+    },
+    "semantic-version": {
+      "description": "A semantic version string. See https://semver.org/",
+      "type": "string",
+      "pattern": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
+    },
+    "sha1": {
+      "type": "string",
+      "pattern": "[0-9a-fA-F]{40}"
```

Comment:
```suggestion
      "pattern": "^[0-9a-fA-F]{40}$"
```

Reply:
You found a bug! https://github.com/microsoft/vcpkg-ce-catalog/pull/31